### PR TITLE
Updated cc search item in top navigation

### LINKED ISSUE #473: Update CC Search items in top navigation
Currently, we have two pages related to CC Search in very different locations:
- the top level CC Search page: https://opensource.creativecommons.org/cc-search/
- the CC Search contribution guide under "Contributing": https://opensource.creativecommons.org/contributing-code/cc-search/

These are logically related and should be moved to the same place.

We should have a top level "CC Search" item that has two sub pages:
- Overview & Roadmap, that goes to https://opensource.creativecommons.org/cc-search/
- Contribution Guide, that goes to https://opensource.creativecommons.org/contributing-code/cc-search/

This will involve updating the URLs, so the previous URLs should redirect to the new pages. For an existing example of a redirect, see https://github.com/creativecommons/creativecommons.github.io-source/blob/master/content/gsoc-2019/contents.lr

### PR DESCRIPTION
**Pull request by ooyeraju**
## Fixes

Fixes #473  by @kgodey 

## Description
Made a CC Search dropdown in the navbar that has links to two other pages as mentioned in the issue.

## Screenshots

Screenshot from 2020-08-23 16-59-21

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: themes/vocabulary_theme/templates/layout.html [hunk 1]
```diff
@@ -119,10 +119,18 @@
...
-            href="{{ '/cc-search'|url }}">
-            CC Search
-          </a>
+          <div class="navbar-item has-dropdown is-hoverable">
+            <a class="navbar-link is-arrowless {% if this.is_child_of('/cc-search') %} is-active{% endif%}">
+              CC Search<i class="icon caret-down"></i></a>
+            <div class="navbar-dropdown">
+              {% for href, title in [ 
+                ['/cc-search', 'Overview & Roadmap'],
+                ['/contributing-code/cc-search', 'Contribution Guide'],
```
Comment (kgodey) [inline themes/vocabulary_theme/templates/layout.html:128]: Please use the new URL path for this.

Comment (ooyeraju) [reply themes/vocabulary_theme/templates/layout.html:128]: Ok sure.

Comment (ooyeraju) [reply themes/vocabulary_theme/templates/layout.html]: I have made the changs. Thanks a lot for your guidance @kgodey

**Comment by kgodey** [comment]
Thank you @ooyeraju!

**Pull request merged**